src/TimeGrid.jsx

```diff
@@ -77,9 +80,24 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
 
+    if (this._scrollRatio) {
+      const { content } = this.refs;
+      content.scrollTop = content.scrollHeight * this._scrollRatio;
+      // Only do this once
+      this._scrollRatio = null;
```

Comment:
that won't happen though since `componentDidMount` is only ever called once over the lifecycle of the component.

FYI you don't have any `componentDidUpdate` call here